utils/das_data_sorter.py

```diff
@@ -167,4 +249,8 @@ def write_to_output_file(self, file_output:str) -> None:
 if __name__ == '__main__':
     data = pd.read_csv(args.input) # Loads and reads CSV file
     das_sort = DasSort(data, args.unit) # Filters data in CSV file
+    if args.smooth == "mean": # Applies smoothing technique, when provided
+        das_sort.mean_smooth_caller(args.n)
+    elif args.smooth == "median":
+        das_sort.median_smooth_caller(args.n)
```

Comment:
You asked about whether having two separate functions or a single function to do the smoothening would be better. I think what you've got already is best in this case!

If we had a single function, we'd have to also pass in what type of smoothening we wanted, probably as a string. We'd then have to worry about checking that the string is a valid option, and stuff like case sensitivity in a class that ideally shouldn't be concerned about such stuff. By only having two choices (by calling one of either function), it's impossible to tell `DasSort` to do an invalid smoothening technique.